.github/CONTRIBUTING.md

```diff
@@ -86,9 +88,9 @@ git pull --rebase upstream master
 git push --force-with-lease feature/325-japanese-translation
 ```
 
-### 9. Merging a PR (maintainers only)
+## 9. Merging a PR (maintainers only)
```

Comment:
There is a paragraph just up from here that starts with "To learn more about rebasing in Git..." that has some strange square brackets.  It doesn't seem like they would be links, but maybe they are.  It just needs some TLC of some kind.